seed/views/v3/properties.py

```diff
@@ -1376,6 +1403,17 @@ def update_with_building_sync(self, request, pk):
         )
 
         if p_status and new_pv_state:
+            if at_updated:
+                new_pv_state.extra_data.update({'at_updated_at': at_updated})
+                new_pv_state.save()
+                Column.objects.get_or_create(
+                    is_extra_data=True,
+                    column_name='at_updated_at',
+                    display_name='Audit Template Updated',
+                    organization=cycle.organization,
+                    table_name='PropertyState',
+                )
```

Comment:
update extra data to have at_updated field and create a column if it DNE to allow the frontend to share the value